UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.SceneList.cs

```diff
@@ -73,18 +83,14 @@ IEnumerator ServerLoadMainStation(SubsceneLoadTimer loadTimer)
 	{
 		MainStationLoaded = true;
 		//Auto scene load stuff in editor:
-		var prevEditorScene = GetEditorPrevScene();
-		if (AdminForcedMainStation == "Random" && prevEditorScene.Contains("Lobby") == false && (prevEditorScene != ""))
+		switch (AdminForcedMainStation)
 		{
-			serverChosenMainStation = prevEditorScene;
-		}
-		else if (AdminForcedMainStation == "Random")
-		{
-			serverChosenMainStation = mainStationList.GetRandomMainStation();
-		}
-		else
-		{
-			serverChosenMainStation = AdminForcedMainStation;
+			case "Random":
+				serverChosenMainStation = mainStationList.MainStations.PickRandom();
+				break;
+			default:
+				//serverChosenMainStation = AdminForcedMainStation;
```

Comment:
Admin forcing map commented out. This feels like it's running into the same issue the admin sound player has, if you figure out how to do this then it can be used to fix the sound player as well.

Reply:
Intentional, this will be sorted out in a later PR.